openpdf/src/main/java/com/lowagie/text/pdf/AcroFields.java

```diff
@@ -1089,7 +1090,11 @@ public boolean setFieldProperty(String field, String name, Object value, int[] i
               PdfAppearance cb = new PdfAppearance();
               if (dao[DA_FONT] != null) {
                 ByteBuffer buf = cb.getInternalBuffer();
-                buf.append(new PdfName((String) dao[DA_FONT]).getBytes()).append(' ').append((Float) value).append(" Tf ");
+                Float fontSize = (float) Font.DEFAULTSIZE;
```

Comment:
Maybe as ternery expression?
```java
Float fontSize = value == null ? (float) Font.DEFAULTSIZE : (Float) value;
```

Reply:
It would certainly be shorter but I m not fan of the ternary operator.